__tests__/app.test.js

```diff
@@ -153,4 +150,34 @@ describe("app/articles", () => {
         });
     });
   });
+  describe("POST/api/articles/:article_id/comments", () => {
+    test("200: returns the comment that was just posted to the correct article_id given", () => {
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send({ username: "butter_bridge", body: "who is Mitch?" })
+        .expect("Content-Type", "application/json; charset=utf-8")
+        .expect(201)
+        .then(({ body }) => {
+          const addedComment = body.newComment[0];
+          expect(addedComment).toMatchObject({
+            author: "butter_bridge",
+            body: "who is Mitch?",
+            article_id: 3,
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+    });
+    test("400: returns a bad request error when the information needed to add a new comment is incomplete", () => {
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send({ username: "butter_bridge" })
+        .expect("Content-Type", "application/json; charset=utf-8")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Bad Request");
+        });
+    });
```

Comment:
add another 400 test for an invalid id eg /api/articles/banana/comments
add a 404 test for a non existent id eg /api/articles/1000/comments
add a 404 test for a non existent username